shelley/chain-and-ledger/executable-spec/src/LedgerState.hs

```diff
@@ -1146,12 +1148,38 @@ createRUpd (BlocksMade b) (EpochState acnt ss ls pp) =
         eta = fromIntegral blocksMade / expectedBlocks
 
 -- | Overlay schedule
+-- This is just a very simple round-robin, evenly spaced schedule.
+-- The real implementation should probably use randomization.
 overlaySchedule
-  :: Set (VKeyGenesis dsignAlgo)
+  :: Epoch
+  -> Set (VKeyGenesis dsignAlgo)
   -> Seed
   -> PParams
   -> Map.Map Slot (Maybe (VKeyGenesis dsignAlgo))
-overlaySchedule = undefined
+overlaySchedule e gkeys _ pp = Map.union active inactive
```

Comment:
the change of the type signature should also be in the formal spec definition of `overlaySchedule`

Reply:
yea, I'll do that right away